File: app/src/main/java/msr/mirudl/app/UiHelper.java
package msr.mirudl.app;

import android.graphics.drawable.GradientDrawable;
import android.graphics.drawable.Drawable;

public final class UiHelper {

    private UiHelper() {}

    public static Drawable rounded(int color, int radiusDp) {
        GradientDrawable drawable = new GradientDrawable();
        drawable.setShape(GradientDrawable.RECTANGLE);
        drawable.setColor(color);
        drawable.setCornerRadius(radiusDp);
        return drawable;
    }
}

File: app/src/main/java/msr/mirudl/app/UiHelper.kt
package msr.mirudl.app

import android.graphics.drawable.GradientDrawable

object UiHelper {

    @JvmStatic
    fun rounded(color: Int, radiusDp: Int): GradientDrawable {
        return GradientDrawable().apply {
            shape = GradientDrawable.RECTANGLE
            setColor(color)
            cornerRadius = radiusDp.toFloat()
        }
    }
}
